src/tracking.c

```diff
@@ -361,7 +362,7 @@ void trackingInvalidateKeysOnFlush(int dbid) {
         while ((ln = listNext(&li)) != NULL) {
             client *c = listNodeValue(ln);
             if (c->flags & CLIENT_TRACKING) {
-                sendTrackingMessage(c,"",1,0);
+                sendTrackingMessage(c,shared.null[c->resp]->ptr,sdslen(shared.null[c->resp]->ptr),1);
```

Comment:
This will need to be at a lower level. Right now you're just passing binary representation of a null as the keyname (e.g. "-1\r\n"), which is a valid. You'll need to call addReply(shared.null[c->resp]) directly somewhere, probably in sendTrackingMessage() if the keyname is NULL.

Reply:
Thank you for looking @madolson!

Could you help me understand a bit more about what I've gotten wrong here? Based on the function docs I would have expected that when specifying the 4th argument of this function as '1' that a preformatted message would be allowed in the 2nd argument. If that's not right and you can tell me a bit more, then I could also update function docs for sendTrackingMessage.

Reply:
My bad, I missed that you updated that the last field. So I think this change looks good.